Sprint-1/exercises/random.js

```diff
@@ -7,3 +7,11 @@ const num = Math.floor(Math.random() * (maximum - minimum + 1)) + minimum;
 // Try breaking down the expression and using documentation to explain what it means
 // It will help to think about the order in which expressions are evaluated
 // Try logging the value of num and running the program several times to build an idea of what the program is doing
+
+// Math.random(): This method generates a random floating-point number between 0 and 1 but it will never actually be 1.
```

Comment:
If you want to find an amazing resource for all things JavaScript, you can always rely on MDN documentation. [Here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random) you can read more about how random works in JavaScript. I remember when I first got introduced to Math.random library in JavaScript, I was a bit confused to how it worked (Yes, JavaScript is very unique in the way it does things) :) 

I find reading MDN documentation is a great way to stay informed and be up-to-date with latest JavaScript updates & changes!

Reply:
Absolutely, MDN (Mozilla Developer Network) documentation is a fantastic resource for learning JavaScript and staying up-to-date with the latest updates and changes. thanks for your suggestion